full-v3.yml

```diff
@@ -1870,14 +1950,6 @@ definitions:
         - "Y"
         - "N"
         - ""
-      home_language_name:
-        type: string
-        x-nullable: true
-        x-validation: true
```

Comment:
This is confirmed that we want to remove this?

Reply:
Yes. We're packing the `home_language` field with the name of the language so this field is redundant. 
The enum of `home_language` is different for v3.0 and v3.1 if you look at the changes to `v3/generate.go`'s `modifyDefinitions` function.
`home_language_name` was introduced during an intern project in Summer '22.

Reply:
Attaching a [link here of our demographic fields](https://docs.google.com/spreadsheets/d/15oeoRLfpWYJmxEhDl978I0j-o39WJY-5rsp49LzTqaI/edit#gid=0) for this project for your convenience.